tests/mod.rs

```diff
@@ -67,6 +67,33 @@ async fn trace_with_tokio_attribute() {
 }
 
 
+#[cfg(feature = "trace")]
+#[test_env_log::test]
+fn test_tracing_error() {
+  use tracing::{span, Level};
+  use tracing_error::SpanTrace;
+
+  let captured_inner_span;
+  let captured_outer_span;
+
+  {
+    let outer_span = span!(Level::ERROR, "outer");
+    let _guard = outer_span.enter();
+    {
+      let inner_span = span!(Level::ERROR, "inner");
+      let _guard = inner_span.enter();
+        captured_inner_span = SpanTrace::capture();
+    }
+    captured_outer_span = SpanTrace::capture();
+  }
+  assert!(format!("{}", captured_inner_span).contains("inner"));
```

Comment:
Maybe it is enough to assert that the spans are collected,
but this way test also shows the behaviour of spans which might have educational value